starknet_py/tests/e2e/account/account_test.py

```diff
@@ -121,6 +121,8 @@ async def test_rejection_reason_in_transaction_receipt(account, map_contract):
     transaction_receipt = await account.client.get_transaction_receipt(res.hash)
 
     if isinstance(account.client, GatewayClient):
+        assert "Actual fee exceeded max fee." in transaction_receipt.revert_error
```

Comment:
Should it not be rejection_reason for both?
I thought revert_error is for reverted transactions

Reply:
I may have misunderstood that field then 😅 , will investigate that further.